File: Android/sources/com/android/billingclient/api/zzan.java
package com.android.billingclient.api;

public final /* synthetic */ class zzan implements Runnable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ ExternalOfferAvailabilityListener zzb;

    public /* synthetic */ zzan(BillingClientImpl billingClientImpl, ExternalOfferAvailabilityListener externalOfferAvailabilityListener) {
        this.zza = billingClientImpl;
        this.zzb = externalOfferAvailabilityListener;
    }

    public final void run() {
        this.zza.zzX(this.zzb);
    }
}

File: IOS/v2/com/android/billingclient/api/zzan.swift
package com.iOS.billingclient.api;.swift
public final /* synthetic */ class zzan implements Runnable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ ExternalOfferAvailabilityListener zzb;.swift
public /* synthetic */ zzan(BillingClientImpl billingClientImpl, ExternalOfferAvailabilityListener externalOfferAvailabilityListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = externalOfferAvailabilityListener;.swift
}.swift
public final void run() {.swift
this.zza.zzX(this.zzb);.swift
}.swift
}.swift
